Bullwinkle.class.nut

```diff
@@ -162,11 +165,18 @@ class Bullwinkle {
         if (_watchdogTimer == null) _watchdog();
 
         // Increment the tries
-        if (message.type == BULLWINKLE_MESSAGE_TYPE.SEND) {
-        	message.tries++;
+        if (message.type == BULLWINKLE_MESSAGE_TYPE.SEND && message.id in _packages) {
+		        _packages[message.id]._tries++;
```

Comment:
Sorry - copy/paste from the IDE to Atom does some weird things...  Getting ElectricImp-Sublime working is a must! :)